config-files/usr/share/plasma/desktoptheme/openSUSE/widgets/branding.svgz

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Is a plain SVG file, but with `svgz` extension.

Reply:
The file was renamed per your request.

Reply:
So, originally I did not want to deviate too much from the themes that are currently on opensuse tumbleweed. Most of them are heavily customized compared to breeze variants. 
I thought to just tweak the minimum to get it improved. But if you prefer to have a major overhaul of those and bring opensuse themes more in line with breeze, I am happy to help and will do my best.

Reply:
> I thought to just tweak the minimum to get it improved. But if you prefer to have a major overhaul of those and bring opensuse themes more in line with breeze, I am happy to help and will do my best.

If the theme is meant to be enabled by default, this is IMO a necessary precondition. For non-default themes it doesn't matter as much.

Reply:
I have no issue at all with that. I did not create those themes so I did not want to butcher other people's work too much :)
So if I remember well we already fallback to breeze dark or breeze light on those 3 themes. Means if no item is found, fallback theme is used instead. Does it mean we can basically remove most of the icons and widgets and lonly eave the branding plus start icon?

Reply:
The branding and start icons are already injected into the breeze and oxygen icon themes (`/usr/share/icons/breeze/places/16/start-here-branding.svg`) so not part of the theme or lnf.

Reply:
Maybe we could also adjust the highlight/accent colours, considering Plasma guys want to make that easier for the users anyway https://phabricator.kde.org/T11052

Reply:
And I mean breeze green, not openSUSE green, openSUSE green really doesn't work well with breeze

Reply:
@Vogtinator @hellcp  I made all the modifications, left a very small set of files for color schemes, metadata and start icon in svg format. I will upload a few screenshots and from those you can also see the themes folder structure is very clean. Please let me know if it looks ok.
@guoyunhe your patch was not removed. Your color changes still apply.

Reply:
openSUSE theme.
![openSUSE-breezedefaults](https://user-images.githubusercontent.com/6520987/74707239-3188c980-5254-11ea-9f88-04970f97e78d.jpg)

openSUSE dark theme.
![openSUSEdark-breezedefaults](https://user-images.githubusercontent.com/6520987/74707272-4ebd9800-5254-11ea-9605-1c5deec9e3dc.jpg)

openSUSE light theme.
![openSUSElight-breezedefaults](https://user-images.githubusercontent.com/6520987/74707296-5c731d80-5254-11ea-9fce-b852eb2f81b3.jpg)